### PR TITLE
[Feature] Updates profile document

### LINKED ISSUE #7738: ✨ Improvements to printed profile - including contact info
## Description
This issue addresses comments from the recruitment team about the printed profiles:
- Adding the full contact information into the profile

## 🕵️ Details

1. Add a new section "Contact information" at the top of the printed profile (just below users name) with Email, Phone, Current City, Province and all 3 language fields from the Personal and contact information section of the new profile.

Image

2. Rename the download link to "Print profile"

3. Add another download link (beside the existing one) with text "Print profile (anonymized)"

### 4. Update printed profile with contact info

Add contact info to current printed profile sections  so we can have a complete profile when downloading

### 5. New partial profile download option

To-do list for new anonymized variation of the printed profile:
- The name of the user should only be displayed as [First name] [first letter of Last Name]
- Remove the contact information section so the profile can not be identified easily
- Update the button in the "Admin - User profile" page to include new download variant (single download)
- Update the buttons at the bottom of any candidate table to include new download variant (batch download)

#### SCREENSHOTS 

<details><summary>Download interaction on Admin - User profile</summary>
<p>

image

</p>
</details> 

<details><summary>Download interaction on Admin - Pools - Manage candidates</summary>
<p>

image

</p>
</details> 

[Link to figma file for profile update](https://www.figma.com/file/gqC6F2F1WbBxZMsFCiOkF0/Admin---User-Profile?type=design&node-id=401%3A3593&mode=design&t=tBSdmgpdfFgJOcdX-1)

[Link to figma for table update](https://www.figma.com/file/zUSn85QtLHbhtjlBhdfT6C/Pools---Record-of-Decision-(Copy)?type=design&node-id=5115%3A12979&mode=design&t=E6NHuPeg7RXL2FZL-1)

## 🎨 Design File

https://docs.google.com/document/d/13epIYbnfFdYc-T1_d8SqCKhaJcAVJEXDYkhGBmzLLdA/edit#heading=h.picugz4qrg9l

### 🧑‍🎨 Designer

@Jerryescandon 

## ✅ Acceptance Criteria

- [ ] Add new download link and rename old one
- [ ] Add new section "Contact information"

### PR DESCRIPTION
**Pull request by mnigh**
🤖 Resolves #7738 

## 👋 Introduction

- Adds new "Contact information" section to print profile document.
- Creates a dropdown button which has two options for admins: With all information, with unidentified information.
- Should be able to print 1 or more profile documents with either option.

## 🕵️ Details

- Needs final review from Jerbo on text and styling.

## 🧪 Testing

1. Login as admin and go to admin dashboard.
2. Try out button with one or more users on all pages: User Table, Pool Candidates table, User profile.
3. Ensure profile documents are showing correct information depending on which option is chose.
4. Ensure only [First name] [First letter of last name]. is shown on unidentified option, and no contact information is shown.
5. Ensure full name and contact information is shown on "all information" option.

### CODE REVIEW

File: apps/web/src/components/ProfileDocument/ProfileDocument.tsx [hunk 1]
```diff
@@ -187,6 +189,89 @@ const ProfileDocument = React.forwardRef<HTMLDivElement, ProfileDocumentProps>(
...
+                              {intl.formatMessage(commonMessages.dividingColon)}
+                              {intl.formatMessage(
+                                getLanguage(result.preferredLang),
+                              )}
+                            </p>
+                          )}
+                          {result.preferredLanguageForInterview && (
+                            <p>
+                              {intl.formatMessage({
+                                defaultMessage: "Spoken Interview language",
```
Comment (petertgiles) [inline apps/web/src/components/ProfileDocument/ProfileDocument.tsx:247]:
```suggestion
                                defaultMessage: "Spoken interview language",
```

File: apps/web/src/components/ProfileDocument/ProfileDocument.tsx [hunk 2]
```diff
@@ -187,6 +189,89 @@ const ProfileDocument = React.forwardRef<HTMLDivElement, ProfileDocumentProps>(
...
+                                getLanguage(
+                                  result.preferredLanguageForInterview,
+                                ),
+                              )}
+                            </p>
+                          )}
+                          {result.preferredLanguageForExam && (
+                            <p>
+                              {intl.formatMessage({
+                                defaultMessage: "Written Exam language",
```
Comment (petertgiles) [inline apps/web/src/components/ProfileDocument/ProfileDocument.tsx:263]:
```suggestion
                                defaultMessage: "Written exam language",
```

File: apps/web/src/pages/Users/AdminUserProfilePage/components/UserProfilePrintButton.tsx [hunk 3]
```diff
@@ -1,46 +1,106 @@
...
-  children,
-  anonymous,
+  users,
+  color,
+  mode,
+  beforePrint,
 }: UserProfilePrintButtonProps) => {
   const intl = useIntl();
 
+  const [isAnonymous, setAnonymous] = useState("");
```
Comment (petertgiles) [inline apps/web/src/pages/Users/AdminUserProfilePage/components/UserProfilePrintButton.tsx:32]: Could you constrain the type on this?  Probably a string union?

**Review by petertgiles — APPROVED** [review]
Works well for me!  :1st_place_medal:   I've left a few small things you could touch up.  Could you get a review from @Jerryescandon as well?

File: apps/web/src/components/ProfileDocument/ProfileDocument.tsx (hunk 1 above)
Comment (yonikid15) [reply apps/web/src/components/ProfileDocument/ProfileDocument.tsx]: oops haha fixed by [cb6ac55](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7749/commits/cb6ac55b0ab8a3abd4ca94293bdd23f8e019190e)

File: apps/web/src/components/ProfileDocument/ProfileDocument.tsx (hunk 2 above)
Comment (yonikid15) [reply apps/web/src/components/ProfileDocument/ProfileDocument.tsx]: Fixed by [cb6ac55](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7749/commits/cb6ac55b0ab8a3abd4ca94293bdd23f8e019190e)

File: apps/web/src/pages/Users/AdminUserProfilePage/components/UserProfilePrintButton.tsx (hunk 3 above)
Comment (yonikid15) [reply apps/web/src/pages/Users/AdminUserProfilePage/components/UserProfilePrintButton.tsx]: Sure thing 👍 Fixed by [8f8c9c7](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7749/commits/8f8c9c7aa60474fc9c5213646d708f5e0cc4e7c1)

**Pull request merged**